frdrpc/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.16.2-buster
+
+RUN apt-get update && apt-get install -y \
+  git \
+  protobuf-compiler='3.6.1*' \
+  clang-format='1:7.0*'
+
+# We don't want any default values for these variables to make sure they're
+# explicitly provided by parsing the go.mod file. Otherwise we might forget to
+# update them here if we bump the versions.
+ARG PROTOC_GEN_VERSION
+ARG GRPC_GATEWAY_VERSION
+
+ENV FALAFEL_VERSION="v0.7.1"
```

Comment:
`FALAFEL_VERSION` unnecessary?

Reply:
Correct, obviously this isn't needed :sweat_smile: